### PR TITLE
[FIX] sale: prevent wrong compute fiscal position

### PR DESCRIPTION
**Pull request by yosa-odoo**
Steps to reproduce:
- Create a FP, autodetect (usa, new-york)
- Create a FP with no auto-detect
- Create a customer with: United-states - New-york
- Create an invoice, with the newly created customer
- In "Other info", change the FP to the no-auto-detec
- Confirm

Issue:
- the fp changed to the auto-detect one

opw-2984396

### CODE REVIEW

**Review by agr-odoo — COMMENTED** [review]
Giving a look at the history this file it received several edits "recently"
Examining the most relevant for this PR:
In 48401aa9a0d40db1e0711063ed27e6a9f761e89e the `_onchange_partner_shipping_id` (override of [the one defined in account](https://github.com/odoo/odoo/blob/f4198e497fba8d22231ce22ea148b73e6e207d4c/addons/account/models/account_move.py#L642)) was replaced in favor of a computed `fiscal_position_id`
In 3732aea4b884eb3fadec344c877ebc5763bdbb40 the compute was simplified, but now it looks identical to the base `_onchange_partner_shipping_id`

In conclusion, as it seems the `fiscal_position_id` override is only complicating the computation, we may remove it

:+1:  for the test

**Comment by yosa-odoo** [comment]
Overriding it completely seems to be problematic. I just used a `pass`

**Comment by agr-odoo** [comment]
@smetl 
Hello, this override of `fiscal_position_id` seems no longer useful, could you double check this?

**Review by smetl — APPROVED** [review]
@Feyensv Does it mean you approved this changes or you don't care?

Nobody is currently working on 15.2 except odoo.com and the fiscal position is now a computed field since 16.0 so I think it can be removed.

**Comment by smetl** [comment]
@robodoo r+
